_locales/en/messages.json

```diff
@@ -1284,9 +1281,6 @@
 	"tooltip_restrict_on": {
 		"message": "Trackers blocked and Anti-Tracking enabled on site. Click to Undo."
 	},
-	"tooltip_custom_settings": {
```

Comment:
I'm seeing that this token is also used in `GhosteryFeature.jsx`

Reply:
The token in `GhosteryFeature.jsx` was in some commented out code that I had left in there and which is now removed. The remaining code does not use this token.